tests/test_algorithm.py

```diff
@@ -425,6 +425,23 @@ def test_asset_lookup(self):
         self.assertIsInstance(algo.sid(3), Equity)
         self.assertIsInstance(algo.sid(4), Equity)
 
+        # Supplying a non-string argument to future_symbol()
+        # should result in a TypeError.
+        with self.assertRaises(TypeError):
+            algo.symbol(1)
+
+        with self.assertRaises(TypeError):
+            algo.symbol((1,))
+
+        with self.assertRaises(TypeError):
+            algo.symbol(set(1,))
+
+        with self.assertRaises(TypeError):
+            algo.symbol(list(1,))
```

Comment:
`set(1,)` and `list(1,)` definitely take me longer to parse than `{1}` and `[1]`.  Any reason for the former?